pyomo/contrib/mindtpy/config_options.py

```diff
@@ -840,22 +842,6 @@ def _add_roa_configs(CONFIG):
             description='The solution limit for the regularization problem since it does not need to be solved to optimality.',
         ),
     )
-    CONFIG.declare(
-        'reduce_level_coef',
-        ConfigValue(
-            default=False,
-            description='Whether to reduce level coefficient in ROA single tree when regularization problem is infeasible.',
-            domain=bool,
-        ),
-    )
-    CONFIG.declare(
-        'use_bb_tree_incumbent',
-        ConfigValue(
-            default=False,
-            description='Whether to use the incumbent solution of branch & bound tree in ROA single tree when regularization problem is infeasible.',
-            domain=bool,
-        ),
-    )
     CONFIG.declare(
```

Comment:
The two options above are removed?

Reply:
In RLP/NLP algorithm, the regularization problem might become infeasible. There were two options in this case. The first is to use the incumbent solution. Another is to reduce the level coefficient for the trust region. In our paper, we only mention the first method. The implementation of reducing the level coefficient makes the code a little complicated. Therefore I just removed these two configs and use the incumbent solution by default.
<img width="940" alt="image" src="https://github.com/Pyomo/pyomo/assets/15854830/8917973c-bbfa-4b7e-a6c9-bc1b5a5c3d1b">